kata-containers/tests
# # Kata Containers 3.0.0-alpha0

- metrics: Enable write io and bw for FIO metrics
- merge runtime-rs to main
- metrics: Use clean_env_ctr function in memory script
- tracing: Add checks for specific spans and ordering
- metrics: Enable read percentile fio metrics
- metrics: delete space in command string
- metrics: Delete iperf3 server and client pods
- common: Improve ctr cleanup
- metrics: Enable latency network measurement
- runk: add tests for runk
- ci: Add generic user agent to check_url
- agent-ctl: Install agent-ctl in install_runtime.sh
- ci: set SEV TEE_TYPE
- metrics: Add latency network metrics script
- metrics: Enable FIO read tests
- ci: Fix tests that are failing with cgroupsv2
- nydus: upgrade nydus version
- metrics: Add script to run FIO in metrics CI
- fc: fix uncompressed dir path
- k8s: Add an integration test for checking process leak
- metrics: Increase boot time value for qemu
- ci: Remove duplicated kubernetes flag
- versions: Update xurls version
- metrics: Add asynchronous job FIO configuration
- makefile: Add rule for fio test for kata CI
- ci: Remove duplicated kubernetes flag
- tests: Skip nydus integration test
- metrics: Support sync engine configuration for FIO tests
- build: Make test into several sub-tests
- ci: Add support for ubuntu 22.04
- Dragonball: add dragonball to check-spelling dictionary
- build: Enable QEMU cache for s390x
- spell-check: Add 'rustlang' to spell check dictionary
- ci: change commit body line limit from 72 to 150
- metrics: Create a common for kubernetes

4f8b2dd3 metrics: Enable write io and bw for FIO metrics
58c7ec03 ci: change privilege for runtime-rs
b3d239a9 ci: check if VC_POD_DIR exists before checking the files in VC_POD_DIR
416f2e70 ci: add sudo when compile runtime-rs and enable k8s-seccomp.bats
c5fc41da ci: install rust for runtime-rs
2416ed7e ci: support CI test for dragonball hypervisor
be473364 ci: build rust runtime for dragonball hypervisor
c162ca38 metrics: Enable read percentile fio metrics
9d5da8cc metrics: Use clean_env_ctr function in memory script
3824d75f metrics: Delete iperf3 server and client pods
065b16ca common: Improve ctr cleanup
78460da2 metrics: delete space in command string
1841269a ci: Add generic user agent to check_url
f7be8907 tracing: Add checks for specific spans and ordering
29e4f2f0 metrics: Enable latency network measurement
f648b662 runk: add tests for runk
6d969992 agent-ctl: Install agent-ctl in install_runtime.sh
5c4da516 metrics: Add latency network metrics script
5ad7ec7c metrics: Enable FIO read tests
2389ea0a ci: Fix tests that are failing with cgroupsv2
c63188fd ci: Remove duplicated kubernetes flag
2d46e2d4 k8s: Add an integration test for checking process leak
01027e8c fc: fix uncompressed dir path
586060a4 metrics: Add script to run FIO in metrics CI
bad8575d metrics: Increase boot time value for qemu
e8b2b2df nydus: upgarde nydus version
d75ce650 versions: Update xurls version
8aecb458 makefile: Add rule for fio test for kata CI
ead50d68 tests: Skip nydus integration test
555a94db metrics: Add asynchronous job FIO configuration
f83d5348 ci: set SEV TEE_TYPE
08b18caa ci: Remove duplicated kubernetes flag
d7041668 build: Make test into several sub-tests
df807372 metrics: Support sync engine configuration for FIO tests
040bae2f Dragonball: add dragonball to check-spelling dictionary
ba1dd852 ci: Add support for ubuntu 22.04
5c0164e9 build: Enable QEMU cache for s390x
93b183b4 spell-check: Add 'rustlang' to spell check dictionary
46edd178 ci: change commit body line limit from 72 to 150
823102fa metrics: Create a common for kubernetes
